### PR TITLE
feat: Media Card

### PR DESCRIPTION
**Pull request by calebpanza**
## ✏️ Description

This PR introduces the Media Card into the UI Kit and migrates the Progress Bar into UI Kit 2.0 as well

[Media Card Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2678%3A21023&t=uSPjAx9dizhmj5iC-4)
[Progress Bar Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2357%3A12468&t=uSPjAx9dizhmj5iC-4)

## 🔬 To Test
1. Set your PROJECT_VARIANT to `storybook`
2. Run storybook and check out the stories

## 📸 Screenshots

| Feature | Simulator | Figma | Links |
| --- | --- | --- | --- |
| Plain | Screenshot_20221221-131946 | Media Card | [Link to Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2716%3A23487&t=uSPjAx9dizhmj5iC-4) |
| With Label | Screenshot_20221221-131857 | Media Card-1 | [Link to Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2716%3A23487&t=uSPjAx9dizhmj5iC-4) | The reason that we chose to do... |
| With Progress Bar | Screenshot_20221221-131919 | Media Card-2 | [Link to Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=2716%3A23487&t=uSPjAx9dizhmj5iC-4) | The reason that we chose to do... |
| With Label and Progress Bar | Screenshot_20221221-131929 | Media Card-3 | [Link to Figma](linkurl) | The reason that we chose to do... |

## 📝 Checklist

- [x] I have performed a self-review of my code
- [x] I have tested my code works in the project (nothing breaks)
- [x] I have commented my code in hard-to-understand areas
- [x] New and existing tests and linting pass locally with my changes
- [x] I have made corresponding changes to the documentation (new envs?)
- [x] For PR's with design updates/changes, I have consulted with the product designer and made all necessary adjustments in Figma

### CODE REVIEW

**Comment by redreceipt** [comment]
I'm recusing myself. This should probably require a designer to sign off.

**Comment by conrad-vanl** [comment]
This looks good from a strictly design perspective. Still will need a code review

**Comment by jordanwade** [comment]
@calebpanza I think the progress bar is messed up in the stories.
image
image

**Comment by calebpanza** [comment]
@jordanwade fixed! Thanks for catching that

File: templates/mobile/src/ui-kit/components/ProgressBar/__stories__/ProgressBar.stories.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const PaddedView = styled.View({
+  padding: unit(4),
+});
+
+storiesOf('ProgressBar', module).add('Basic Implementation', () => (
+  <ScrollView>
+    <PaddedView>
+      <H3>Progress</H3>
+      <ProgressBar step={0} />
```
[the remark this answers is not in the document]
Comment (conrad-vanl) [reply templates/mobile/src/ui-kit/components/ProgressBar/__stories__/ProgressBar.stories.js:26]: decimal

**Review by jordanwade — APPROVED** [review]
Looks good! 👍

**Pull request merged**